<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddProject</class>
 <widget class="QDialog" name="AddProject">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>410</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>410</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add Project</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/res.qrc">
    <normaloff>:/toolbar/project-new.png</normaloff>:/toolbar/project-new.png</iconset>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>391</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="VL100">
    <item>
     <layout class="QHBoxLayout" name="HL110">
      <item>
       <layout class="QVBoxLayout" name="VL112">
        <item>
         <widget class="QLabel" name="LblDocID">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>&amp;Project ID</string>
          </property>
          <property name="buddy">
           <cstring>lineEdit0proj_id</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="LblTitle">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>&amp;Title</string>
          </property>
          <property name="buddy">
           <cstring>textEdit0proj_title</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="HL111">
        <item>
         <layout class="QVBoxLayout" name="VL1111">
          <item>
           <widget class="QLineEdit" name="lineEdit0proj_id">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>24</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>9</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="textEdit0proj_title">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="VL120">
      <item>
       <layout class="QHBoxLayout" name="HL122">
        <item>
         <widget class="QLabel" name="LblStartDate">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>&amp;Start Date</string>
          </property>
          <property name="buddy">
           <cstring>dateEdit0proj_startdate</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="LblEndDate">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>&amp;End Date</string>
          </property>
          <property name="buddy">
           <cstring>dateEdit0proj_enddate</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="HL121">
        <item>
         <widget class="QDateEdit" name="dateEdit0proj_startdate">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>24</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="date">
           <date>
            <year>1900</year>
            <month>1</month>
            <day>1</day>
           </date>
          </property>
          <property name="minimumDateTime">
           <datetime>
            <hour>0</hour>
            <minute>0</minute>
            <second>0</second>
            <year>1900</year>
            <month>1</month>
            <day>1</day>
           </datetime>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dateEdit0proj_enddate">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>24</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="date">
           <date>
            <year>1900</year>
            <month>1</month>
            <day>1</day>
           </date>
          </property>
          <property name="minimumDateTime">
           <datetime>
            <hour>0</hour>
            <minute>0</minute>
            <second>0</second>
            <year>1900</year>
            <month>1</month>
            <day>1</day>
           </datetime>
          </property>
          <property name="minimumDate">
           <date>
            <year>1900</year>
            <month>1</month>
            <day>1</day>
           </date>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="HL130">
      <item>
       <layout class="QVBoxLayout" name="VL132">
        <item>
         <widget class="QLabel" name="LblContName">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>&amp;Contractor Name</string>
          </property>
          <property name="buddy">
           <cstring>lineEdit0proj_contname</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="LblContNo">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Contract &amp;Number</string>
          </property>
          <property name="buddy">
           <cstring>lineEdit0proj_contno</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="HL131">
        <item>
         <layout class="QVBoxLayout" name="VL131">
          <item>
           <widget class="QLineEdit" name="lineEdit0proj_contname">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>24</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="placeholderText">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit0proj_contno">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>24</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>9</pointsize>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>285</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="HL101">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QGroupBox" name="GB2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>93</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="LblTotalRecords">
       <property name="geometry">
        <rect>
         <x>4</x>
         <y>0</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>65</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>0 record(s)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="GB1">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>246</y>
     <width>126</width>
     <height>68</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>68</height>
    </size>
   </property>
   <property name="title">
    <string>Records Navigation</string>
   </property>
   <widget class="QLabel" name="LblRecNo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>65</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>65</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>&amp;Rec #</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="buddy">
     <cstring>lineEditRecordNo</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditRecordNo">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>40</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>40</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonLast">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>15</y>
      <width>26</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resources/res.qrc">
      <normaloff>:/navigate/control-stop-right-icon.png</normaloff>:/navigate/control-stop-right-icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonNext">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>15</y>
      <width>26</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resources/res.qrc">
      <normaloff>:/navigate/control-double-right-icon.png</normaloff>:/navigate/control-double-right-icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPrev">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>15</y>
      <width>26</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resources/res.qrc">
      <normaloff>:/navigate/control-double-left-icon.png</normaloff>:/navigate/control-double-left-icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonFirst">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>15</y>
      <width>26</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resources/res.qrc">
      <normaloff>:/navigate/control-stop-left-icon.png</normaloff>:/navigate/control-stop-left-icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>260</y>
     <width>261</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonAddMore">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>95</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>&amp;Save &amp;&amp; Add More</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonCommitAll">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>95</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>&amp;Commit All</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit0proj_id</tabstop>
  <tabstop>textEdit0proj_title</tabstop>
  <tabstop>dateEdit0proj_startdate</tabstop>
  <tabstop>dateEdit0proj_enddate</tabstop>
  <tabstop>lineEdit0proj_contname</tabstop>
  <tabstop>lineEdit0proj_contno</tabstop>
  <tabstop>pushButtonAddMore</tabstop>
  <tabstop>pushButtonCommitAll</tabstop>
  <tabstop>lineEditRecordNo</tabstop>
  <tabstop>pushButtonFirst</tabstop>
  <tabstop>pushButtonPrev</tabstop>
  <tabstop>pushButtonNext</tabstop>
  <tabstop>pushButtonLast</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
